modules/apps/portal/portal-upload-servlet-request-filter/src/main/java/com/liferay/portal/upload/servlet/request/filter/internal/UploadServletRequestFilter.java

```diff
@@ -103,15 +101,12 @@ public void processFilter(
 
 				fileSizeThreshold = portlet.getMultipartFileSizeThreshold();
 				location = portlet.getMultipartLocation();
-				maxRequestSize = portlet.getMultipartMaxRequestSize();
```

Comment:
Is it safe to do this? Are the values from the portlet the same as those coming from `DLValidator`?

Reply:
Yes, I believe this is safe. These values are set based on the model, in the case of DL, these should be the same as DLValidator.